transformations/src/main/java/com/commit451/coiltransformations/BlurTransformation.kt

```diff
@@ -72,25 +73,6 @@ class BlurTransformation @JvmOverloads constructor(
         return output
     }
 
-    override fun equals(other: Any?): Boolean {
```

Comment:
It's not strictly required, but I'd recommend having `equals`/`hashCode` implementations for each transformation in case they're used in Compose (could add them in a separate PR). Without them:

```kotlin
ImageRequest.Builder(context)
    .data(...)
    .transformation(BlurTransformation())
    .build()
```

will have a different `hashCode` on each composition which will retry the request. It'll almost always return the image from the memory cache, but it's less efficient than if the `hashCode` doesn't change.

Reply:
Gotcha, I will do this as a follow up 👍